strawberry/types/info.py

```diff
@@ -15,7 +17,9 @@ class Info(Generic[ContextType, RootValueType]):
     context: ContextType
     root_value: RootValueType
     variable_values: Dict[str, Any]
+    return_type: Optional[Union[Type, StrawberryUnion]]
```

Comment:
this will change to `StawberryType`, right @BryceBeagle? shall we add a comment for it?

Reply:
That's correct.

A comment is a good idea. We may also want to define a `TypeAlias` until `StrawberryType` is implemented.

Reply:
added a comment, is this what you meant?

Reply:
yup!